JustSaying/JustSayingFluently.cs

```diff
@@ -75,12 +75,11 @@ public virtual INamingStrategy GetNamingStrategy()
             var snsWriteConfig = new SnsWriteConfiguration();
             configBuilder?.Invoke(snsWriteConfig);
 
-            _subscriptionConfig.Topic = typeof(T).ToTopicName();
             var namingStrategy = GetNamingStrategy();
 
             Bus.SerialisationRegister.AddSerialiser<T>(_serialisationFactory.GetSerialiser<T>());
 
-            var topicName = namingStrategy.GetTopicName(_subscriptionConfig.BaseTopicName, typeof(T));
+            var topicName = namingStrategy.GetTopicName(snsWriteConfig.BaseTopicName, typeof(T));

```

Comment:
I've always found the whole `Base...Name` stuff a bit weird, so excuse my ignorance. The value we are no longer using (`_subscriptionConfig.BaseTopicName`) used to be populated by `WithSqsTopicSubscriber` which took an optional topic name, how does that play with this after the change?

Reply:
The behaviour of `WithSqsTopicSubscriber` remains the same with this change as `_subscriptionConfig.BaseTopicName` will continue to be used when configuring a topic subscriber (https://github.com/justeat/JustSaying/blob/master/JustSaying/JustSayingFluently.cs#L339)